lib/controllers/situations.js

```diff
@@ -68,3 +68,74 @@ exports.openfiscaResponse = function(req, res, next) {
         res.send(result);
     });
 };
+
+function createTestContent(situation, test) {
+    var openfiscaRequest = mapping.buildOpenFiscaRequest(situation, true);
+    var testCase = openfiscaRequest.scenarios[0].test_case;
+    return _.assign(test, {
+        period: openfiscaRequest.scenarios[0].period,
+        absolute_error_margin: 10,
+        individus: testCase.individus,
+        familles: testCase.familles[0],
+        foyers_fiscaux: testCase.foyers_fiscaux[0],
+        menages: testCase.menages[0]
+    });
+}
+
+function myToString(obj) {
+    return obj.toString();
+}
+
+function normalizeTestContent(testFileContent) {
+    testFileContent.individus = testFileContent.individus.map(function(individu) {
+        individu.id = individu.id.toString();
+        return individu;
+    });
+
+    function replaceElementsWithStringRepresentation(obj) {
+        function convert(leafName) {
+            obj[leafName] = obj[leafName].map(myToString);
+        }
+        return convert;
+    }
+
+    var sources = {
+        familles: ['parents', 'enfants'],
+        foyers_fiscaux: ['declarants', 'personnes_a_charge'],
+        menages: ['enfants'],
+    };
+
+    for (var parent in sources) {
+        sources[parent].forEach(replaceElementsWithStringRepresentation(testFileContent[parent]));
```

Comment:
This is pretty hard to understand, because there are many different logics going on:

1. Imperative iteration: `for in`.
2. Functional iteration: `forEach`.
3. Dynamic closures, since `replaceElementsWithStringRepresentation` actually yields a closure, despite being called as if it would directly replace the elements, which is misleading.

Example refactoring:

```js
    Object.keys(EXPORTED_PROPERTIES).forEach(function(entity) {
        EXPORTED_PROPERTIES[entity].forEach(function(property) {
            testFileContent[entity][property] = testFileContent[entity][property].map(JSON.stringify);
        });
    });
```

(which removes the need for a function that is used in a single place)